plasma_framework/contracts/src/exits/utils/ExitableTimestamp.sol

```diff
@@ -7,21 +7,28 @@ library ExitableTimestamp {
         uint256 minExitPeriod;
     }
 
-    function calculate(
+    function calculateDepositTxOutputExitableTimestamp(
         Calculator memory _calculator,
-        uint256 _now,
-        uint256 _blockTimestamp,
-        bool _isDeposit
+        uint256 _now
     )
         internal
         pure
         returns (uint64)
     {
-        uint256 minExitableTimestamp = _now + _calculator.minExitPeriod;
+        // Please that boosting the exitable timestamp for a deposit should be only done in case of a SE.
```

Comment:
Can you put this on top of the function as the function doc instead?
Also, would be better to link to Kevin's comment directly in that issue.